blackfireio/blackfire-symfony-meta/1.0/post-install.txt

```diff
@@ -0,0 +1,7 @@
+  * Blackfire Agent has been configured in your <comment>docker-compose.yaml</> file
+
+  * Configure your Blackfire <comment>SERVER</> credentials in <info>.env.local</> (get them at https://blackfire.io/my/settings/credentials)
```

Comment:
We should be more explicit here, not to confuse people with environment credentials

Reply:
Thinking about it, we'd better keep this link.
Available credentials (for Blackfire environments and personal server credentials) should probably get listed in the Blackfire settings page.
Ping @ChristopheDujarric